src/Features/CSharp/Portable/ExtractMethod/CSharpSelectionResult.cs

```diff
@@ -68,27 +66,6 @@ protected override SyntaxNode GetNodeForDataFlowAnalysis()
                 : node;
         }
 
-        protected override bool UnderAnonymousOrLocalMethod(SyntaxToken token, SyntaxToken firstToken, SyntaxToken lastToken)
```

Comment:
these are no longer needed as they were hacks on how we previously determined certain syntactic facts about the code being extracted.